pulsar-package-management/bookkeeper-storage/src/main/java/org/apache/pulsar/packages/management/storage/bookkeeper/BookKeeperPackagesStorageConfiguration.java

```diff
@@ -38,8 +38,8 @@ int getPackagesReplicas() {
         return Integer.parseInt(getProperty("packagesReplicas"));
     }
 
-    String getZookeeperServers() {
-        return getProperty("zookeeperServers");
```

Comment:
what happens if you comment out `metadataStoreUrl` from the config and still use `zookeeperServers ` ? 
we have to add a test for backward compatibility

Reply:
it's OK, values in  `BookKeeperPackagesStorageConfiguration.java come from broker and standalone configuration. 
`configuration.getProperty("metadataStoreUrl")` will return either metadataStoreUrl  or zookeeperServers in config by `ServiceConfiguration.getMetadataStoreUrl`. @nicoloboschi